### DISCUSSION TITLE
能否扩展一下get_group_info()接口返回的数据字段

### DISCUSSION THREAD

**Discussion by HuangShiJin1015** [Mrs4s/go-cqhttp#711] [General]
我在调用get_group_info（）接口时，发现返回的数据指向只有四个，分别是：
group_id（群号），group_name（群名称） max_member_count（群规模）member_count（群成员个数）。

能否增加一下群简介，群创建时间和群头像这些数据字段项呢

**Comment by Ink-33** [comment]
#728
